docs/proposals/PROPOSAL_B.md

```diff
@@ -0,0 +1,287 @@
+# Proposal B - Extend Platform Variants and Introduce Compatibility Artifact
+
+This proposal recommends changes to image spec for more granular image selection and introduces a new image compatibility artifact type.
+Based on the artifact, cluster administrators can verify hosts or prepare the appropriate infrastructure for containers.
+
+## Modifications
+
+### Image Spec
+
+Extend the platform variants with new ARM and AMD64 micro-architecture levels.
```

Comment:
Possibly we should extend to a few more? E.g., I'm looking at the keys here: https://github.com/archspec/archspec-json/blob/master/cpu/microarchitectures.json. And minimally we should sanity check across that set to ensure we haven't missed important one(s).

```
dict_keys(['x86', 'i686', 'pentium2', 'pentium3', 'pentium4', 'prescott', 'x86_64', 'x86_64_v2', 'x86_64_v3', 'x86_64_v4', 'nocona', 'core2', 'nehalem', 'westmere', 'sandybridge', 'ivybridge', 'haswell', 'broadwell', 'skylake', 'mic_knl', 'skylake_avx512', 'cannonlake', 'cascadelake', 'icelake', 'sapphirerapids', 'k10', 'bulldozer', 'piledriver', 'steamroller', 'excavator', 'zen', 'zen2', 'zen3', 'zen4', 'ppc64', 'power7', 'power8', 'power9', 'power10', 'ppc64le', 'power8le', 'power9le', 'power10le', 'aarch64', 'armv8.1a', 'armv8.2a', 'armv8.3a', 'armv8.4a', 'armv8.5a', 'armv9.0a', 'thunderx2', 'a64fx', 'cortex_a72', 'neoverse_n1', 'neoverse_v1', 'neoverse_v2', 'm1', 'm2', 'arm', 'ppc', 'ppcle', 'sparc', 'sparc64', 'riscv64', 'u74mc'])
```

Reply:
What about ARM 8.X which is also 64bit? We currently target that in our new ARM port for illumos. That includes Raspberry PI4

Reply:
Soon I will put everything together, including all ARM variants.

Reply:
I've decided to remove it from the proposal.